src/Components/ToDoList/index.js

```diff
@@ -1,15 +1,28 @@
 import { InputField } from "../InputField"
 import { ToDoTask } from "../ToDoTask"
+import {useSelector} from "react-redux"
 
 export function ToDoList ()
 {
+  const tasks = useSelector(state => state.list.tasks)
+  
+  console.log('tasks in list', tasks)
+
+  
+debugger
```

Comment:
your PR should never ever contains console logs and debugger, it should never happen again, always review your PR yourself before sending